src/pybind/mgr/telemetry/module.py

```diff
@@ -272,10 +272,10 @@ def handle_command(self, inbuf, command):
                 r[opt['name']] = getattr(self, opt['name'])
             return 0, json.dumps(r, indent=4), ''
         elif command['prefix'] == 'telemetry on':
-            self.set_config('active', True)
+            self.set_module_option('enabled', True)
```

Comment:
What about using localized function here? Thus it can be enabled/disabled for specific managers.

Reply:
Oops, just saw this.  What's the use case for that?

Reply:
No prob, now i think it's a silly idea which does not make sense.